lib/ansible/modules/windows/win_stat.py

```diff
@@ -175,7 +179,7 @@
             description: the target of the symbolic link, will return null if not a link or the link is broken
             return: success, path exists, file is a symbolic link
             type: string
-            sample: C:\\temp
+            sample: C:\temp
```

Comment:
Fixed. The problem was not that specific part, the whole string was not a raw string (which I thought I fixed in all modules).